js_modules/dagit/packages/core/src/instance/InstanceHealthPage.tsx

```diff
@@ -12,6 +12,12 @@ import {INSTANCE_HEALTH_FRAGMENT} from './InstanceHealthFragment';
 import {InstanceTabs} from './InstanceTabs';
 import {InstanceHealthQuery} from './types/InstanceHealthQuery';
 
+export const DAEMON_LOADING_BOX = (
+  <Box padding={{horizontal: 24}} style={{color: ColorsWIP.Gray400}}>
+    Loading…
+  </Box>
+);
```

Comment:
Not sure this is really complex enough to justify exporting it, even if it's the same in Cloud.